blockmanager.go

```diff
@@ -197,24 +190,14 @@ type blockManager struct { // nolint:maligned
 	wg   sync.WaitGroup
 	quit chan struct{}
 
-	headerList     headerlist.Chain
-	reorgList      headerlist.Chain
-	startHeader    *headerlist.Node
-	nextCheckpoint *chaincfg.Checkpoint
-	lastRequested  chainhash.Hash
-
-	minRetargetTimespan int64 // target timespan / adjustment factor
-	maxRetargetTimespan int64 // target timespan * adjustment factor
-	blocksPerRetarget   int32 // target timespan / target time per block
+	reorgList     headerlist.Chain
+	startHeader   *headerlist.Node
```

Comment:
I would think this information could be folded in as well?

Reply:
I do not understand what you mean by folded in